tests/foreman/ui/test_domain.py

```diff
@@ -95,6 +96,10 @@ def test_update_domain(self, testdata):
         @Assert: Domain is updated
 
         """
+        bug_id = testdata.pop('bugzilla', None)
+        if bug_id is not None and bz_bug_is_open(bug_id):
+            self.skipTest('Bugzilla bug {0} is open.'.format(bug_id))
```

Comment:
What if you use 'skip_if_bug_open' decorator for this test? instead of putting skip in test body.